internal/store/objects/store.go

```diff
@@ -18,26 +18,28 @@ func NewStore(cdc codec.Marshaler, db sdk.KVStore) Store {
 
 // Store builds an object store
 type Store struct {
-	db  sdk.KVStore
-	cdc codec.Marshaler
+	db     sdk.KVStore
+	cdc    codec.Marshaler
+	numObj uint64
 }
 
 // Create creates the object in the store
 // returns an error if it already exists
 // or if marshalling fails
-func (s Store) Create(o types.Object) error {
+func (s *Store) Create(o types.Object) error {
```

Comment:
As I mentioned in Slack, interfaces are passed as pointers, so `*Store` is not proper form, for lack of a better description.  Please revert the receiver back to its non-pointer form.

Reply:
It is reverted to his original form now (as I found another way of doing this) but I'd like to point out that  `Store` in this context is not an interface but a structure defined just above (lines 20-23 of _objects/store.go_)
The package as multiple `Store` definitions, and no interface definition for them (as they do not implement the same methods)

Reply:
I stand corrected.  Thanks.